PortMaster/knulli/control.txt

```diff
@@ -28,12 +28,12 @@ if [ -f "$HOME/.config/gamecontrollerdb.txt" ]; then
 elif [ -f "/tmp/gamecontrollerdb.txt" ]; then
   export SDL_GAMECONTROLLERCONFIG_FILE="/tmp/gamecontrollerdb.txt"
 else
-  export SDL_GAMECONTROLLERCONFIG_FILE="/$controlfolder/batocera/gamecontrollerdb.txt"
```

Comment:
I think that here it's better to point the the one in `knulli` folder even it's the same as in `batocera` folder.